src/components/Item/VisualizationItem/ItemContextMenu/ViewAsMenuItems.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import i18n from '@dhis2/d2-i18n'
+import TableIcon from '@material-ui/icons/ViewList'
+import ChartIcon from '@material-ui/icons/InsertChart'
+import MapIcon from '@material-ui/icons/Public'
```

Comment:
Can these be easily replaced with `ui-icons`?

Reply:
I'm going to do all the icons in a separate branch - didn't want to make this PR too big :)